CMakeLists.txt

```diff
@@ -131,20 +135,17 @@ set(NOVELRT_EXECUTABLE_HEADERS
 )
 
 set(NOVELRT_EXECUTABLE_SOURCES
+  ${NOVELRT_SOURCES}
   src/main.cpp
 )
 
 set(NOVELRT_EXECUTABLE_LINK_LIBRARIES
-  NovelRTLib
+  ${NOVELRT_LINK_LIBRARIES}
```

Comment:
Same, this shouldn't be changed.

Reply:
Same as the last one, not sure what happened here, but will fix.